src/store/tasks.js

```diff
@@ -734,10 +758,21 @@ const actions = {
 			const parent = context.getters.getTaskByUid(task.related)
 			context.commit('deleteTaskFromParent', { task, parent })
 			context.commit('deleteTaskFromCalendar', task)
+			context.commit('cancelTaskDeletion', { task })
+			// If the task is open in the sidebar, close the sidebar
+			// TODO: Also check if a descendant is open
```

Comment:
Not necessary to check for descendants. They will be deleted as well and the sidebar gets closed.